WooCommerce/WooCommerceTests/Extensions/IconsTests.swift

```diff
@@ -194,6 +195,11 @@ final class IconsTests: XCTestCase {
         XCTAssertEqual(starOutlineImage.size, CGSize(width: size, height: size))
     }
 
+    func testStarOutlineImageDefaultSize() {
```

Comment:
I like the idea of naming the test name based on the **behavior**. The name  `testStarOutlineImageDefaultSize` doesn't actually tell me what the expectations are. It's a bit generic and generic test names somehow encourage us to write more assertions in the test which is not related to the original intent. 

This is _probably_ better: 

```suggestion
    func testStarOutlineImageUsesGridiconDefaultSizeByDefault() {
```

With all that said, this is fine for now. 😄 Any test is still better than nothing!

Reply:
Noted!